Assets/StartController.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class StartController : MonoBehaviour
+{
+    // Start is called before the first frame update
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        if (Input.GetMouseButtonDown(0))
```

Comment:
This is fine, but in the future I'd wrap all `if` blocks in `{}` curly braces, just for consistency's sake. :)